<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>156</y>
     <width>94</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/quit.png);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>156</y>
     <width>94</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/ok.png);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="btn_close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pwdLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>97</y>
     <width>58</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Passwd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdlineEdit">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Forum account password</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>195</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/dialog-bg.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>47</y>
     <width>58</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;User&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userlineEdit">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Forum account username</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>8</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ubuntu Kylin forum account  login &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>closeButton</zorder>
  <zorder>okButton</zorder>
  <zorder>btn_close</zorder>
  <zorder>pwdLabel</zorder>
  <zorder>pwdlineEdit</zorder>
  <zorder>userLabel</zorder>
  <zorder>userlineEdit</zorder>
  <zorder>titleLabel</zorder>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>userlineEdit</tabstop>
  <tabstop>pwdlineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
